pkg/webui/console/containers/device-onboarding-form/index.js

```diff
@@ -126,10 +126,9 @@ const DeviceOnboardingForm = () => {
             errors: {},
             values: {
               ...values,
-              ...initialValues,
```

Comment:
Did this break the multi-onboarding? Any idea why?

Reply:
It was overwriting the template values to empty initial values, if I recall correctly.

Reply:
Oh wait. This obviously makes sense, since we want to retain all configs except the IDs.